cypress/lib/helper.js

```diff
@@ -24,6 +12,16 @@ Profile ID: \
 Team member: ionut\
 Status ID:`
 
+export const bookingsList = []
```

Comment:
can we maybe import this data from prisma (seed) and stringify it in order to get the same thing?

Reply:
it does not let me import from "seed.ts". I get Error: Webpack Compilation Error